node/node.go

```diff
@@ -252,6 +252,12 @@ func (n *Node) StartConsensus(epoch uint32) error {
 	}
 
 	bootstrapper.StartSync()
+	epoch := uint32(0)
+	crtBlockHeader := n.blkc.GetCurrentBlockHeader()
+	if !check.IfNil(crtBlockHeader) {
+		epoch = crtBlockHeader.GetEpoch()
+		log.Info("starting consensus", "epoch", epoch)
```

Comment:
shouldn't this log message be displayed in both cases ?

Reply:
:1st_place_medal: done